fta/samples/models.py

```diff
@@ -130,7 +139,19 @@ def __str__(self):
         return self.slug
 
 
+class LabeledElementManager(models.Manager):
+    def get_queryset(self):
+        return (
+            super()
+            .get_queryset()
+            .prefetch_related("labeled_sample")
```

Comment:
The `prefetch_related` calls in the Managers above are for performance optimization, whereas I understand you added this new Manager just to filter out `LabeledElement`s of superseded `LabeledSample`s, right? In which case, we probably don't need the `prefetch_related` here.

Reply:
Removed.